src/WebApiContrib.Core.Formatter.Csv/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.AspNetCore.Mvc.Core": "1.0.0-rc2-final",
+        "NETStandard.Library": "1.5.0-rc2-24027",
+        "System.Reflection.TypeExtensions": "4.1.0-rc2-24027"
+    },
+
+    "frameworks": {
+        "netstandard1.5": {
+            "imports": "dnxcore50"
+        },
+        "net46": {

```

Comment:
like I said before, MVC Core is cross compiled against net451 at the moment, and we wanna support all users. Not sure what you mean by messy reflection, you can use reflection fine while targeting net451, see i.e. https://github.com/aspnet/Mvc/blob/1.0.0-rc2/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/CollectionModelBinderProvider.cs